lib/src/css_mirroring/directives_processor.dart

```diff
@@ -0,0 +1,45 @@
+part of scissors.src.css_mirroring.bidi_css_generator;
+
+class DirectionIndependentDirectivesProcessor extends NodeProcessor {
```

Comment:
These two classes are just glorified functions.

Please just extract process to `processDirectionIndependentDirectives(EditConf...)`
(and same for the other)
They can stay in the same file, but... this file contains 2 processors, so please rename it as well (+= s)